### PR TITLE
Sdk: Structurize relayerConfig

### PR DESCRIPTION
**Pull request by cehali**
# Summary
Add fetchRelayerConfig and change getRelayerConfig to synchronous

Fixes: 0

## Checklist
- [ ] Change is small and easy to review (please split big changes into multiple PRs)
- [ ] Change is consistent with architecture
- [ ] Change is consistent with test architecture
- [ ] Change is consistent with naming conventions
- [ ] New code is covered with tests
- [ ] Tests related to old code are updated
- [ ] Documentation is up to date with changes

### CODE REVIEW

File: universal-login-wallet/test/integration/ui/CreateAccount.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ describe('UI: Creation flow', () => {
         ({relayer, provider} = await setupSdk(wallet, '33113'));
         services = await createPreconfiguredServices(provider, relayer, [ETHER_NATIVE_TOKEN.address]);
         await services.sdk.tokensDetailsStore.fetchTokensDetails();
+        await services.sdk.fetchRelayerConfig();
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/test/integration/ui/CreateAccount.tsx]: I think that we should move these two lines to `createPreconfiguredServices`. 
We can also run just `await sdk.start()`

File: universal-login-sdk/lib/api/sdk.ts [hunk 2]
```diff
@@ -93,7 +93,7 @@ class UniversalLoginSDK {
   }
 
   async createFutureWallet() {
-    await this.getRelayerConfig();
+    await this.fetchRelayerConfig();
```
Comment (marekkirejczyk) [inline universal-login-sdk/lib/api/sdk.ts]: use `getRelayerConfig()` to ensure it is loaded, but don't load it

File: universal-login-wallet/test/integration/helpers/ServicesUnderTests.ts [hunk 3]
```diff
@@ -18,5 +18,7 @@ export const createPreconfiguredServices = async (provider: providers.Provider,
   services.sdk.priceObserver.getCurrentPrices = () => {
     return Promise.resolve({ETH: {USD: 1, DAI: 1, ETH: 1}});
   };
+  await services.sdk.tokensDetailsStore.fetchTokensDetails();
+  await services.sdk.fetchRelayerConfig();
```
Comment (marekkirejczyk) [inline universal-login-wallet/test/integration/helpers/ServicesUnderTests.ts]: Move to sdk.start()

File: universal-login-sdk/lib/api/sdk.ts (hunk 2 above)
Comment (JustynaBroniszewska) [reply universal-login-sdk/lib/api/sdk.ts]: 👍

File: universal-login-wallet/test/integration/helpers/ServicesUnderTests.ts (hunk 3 above)
Comment (JustynaBroniszewska) [reply universal-login-wallet/test/integration/helpers/ServicesUnderTests.ts]: We can use `sdk.start()` here, but we need to remember about:
 - remove `sdk.start()` and `sdk.tokensDetailsStore.fetchTokensDetails()` (lines 31 and 32) [connect.tsx](https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-wallet/test/integration/ui/connect.tsx)
- remove `sdk.tokensDetailsStore.fetchTokensDetails()` (line 26) [CreateAccount.tsx](https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-wallet/test/integration/ui/CreateAccount.tsx)
- add `sdk.stop()` in after in [CreateAccount.tsx](https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-wallet/test/integration/ui/CreateAccount.tsx)
- remove `sdk.tokensDetailsStore.fetchTokensDetails()` (line 16) [setupUI.tsx](https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-wallet/test/integration/helpers/setupUI.tsx)

Comment (cehali) [reply universal-login-wallet/test/integration/helpers/ServicesUnderTests.ts]: We can add two functions to sdk: fetchData, startObserver. Then change sdk.start() to 
```js
start() {
    fetchData();
    startObserver();
}
```

Then in the test, we will use only fetchData where it is necessary.

Comment (cehali) [reply universal-login-wallet/test/integration/helpers/ServicesUnderTests.ts]: @marekkirejczyk

File: universal-login-wallet/test/integration/services/ConfigService.ts [hunk 4]
```diff
@@ -4,24 +4,22 @@ import Relayer from '@universal-login/relayer';
...
   let sdk: UniversalLoginSDK;
   let relayer: Relayer;
-  let relayerConfig: PublicRelayerConfig | undefined;
 
   before(async () => {
-    const [wallet] = await getWallets(createMockProvider());
+    const [wallet] = getWallets(createMockProvider());
     ({sdk, relayer} = await setupSdk(wallet));
-    relayerConfig = await sdk.getRelayerConfig();
+    await sdk.fetchRelayerConfig();
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/test/integration/services/ConfigService.ts]: unnecessary

File: universal-login-web3/test/helpers/createWallet.ts [hunk 5]
```diff
@@ -3,6 +3,7 @@ import UniversalLoginSDK, {DeployedWallet} from '@universal-login/sdk';
 import {ETHER_NATIVE_TOKEN, TEST_GAS_PRICE} from '@universal-login/commons';
 
 export const createWallet = async (name: string, sdk: UniversalLoginSDK, wallet: Wallet) => {
+  await sdk.fetchRelayerConfig();
```
Comment (JustynaBroniszewska) [inline universal-login-web3/test/helpers/createWallet.ts]: This shouldn't be here. We definitely should fetch relayer config in the setup phase.

File: universal-login-wallet/src/core/services/ConfigService.ts [hunk 6]
```diff
@@ -11,7 +11,7 @@ export class ConfigService {
 
   async fetchRelayerConfig() {
```
Comment (xitronix) [inline universal-login-wallet/src/core/services/ConfigService.ts]: Is that service necessary?  We have this functionality in sdk already?

Comment (cehali) [reply universal-login-wallet/src/core/services/ConfigService.ts]: Agree, let's delete it in next PR

**Pull request merged**